scripts/build_windows.bat

```diff
@@ -8,14 +8,15 @@
 setlocal
 
 SET ORIGINAL_DIR=%cd%
-SET CAFFE2_ROOT=%~dp0%..
+SET CAFFE2_ROOT=%~dp0..
+SET PYBIND_DIR="%CAFFE2_ROOT%\third_party\pybind11\include"
```

Comment:
Is this needed for the dedent error is this an unrelated change?

Reply:
No, this was just a pybind11 issue I was having, seemed to fix it but I'm not sure if its just my environment.